sway-ir/src/optimize/simplify_cfg.rs

```diff
@@ -129,7 +134,9 @@ fn merge_blocks(
                     to_block.remove_instruction(context, phi_val);
                 }
             }
-            _otherwise => return Err(IrError::InvalidPhi),
+            _otherwise => {
+                return Err(IrError::InvalidPhi);
+            }
```

Comment:
Are the curlies necessary?

Reply:
> Are the curlies necessary?

Nah, this crept in when I was using a `dbg!` in there. Thanks for catching it :)